src/data/polynomial/laurent.lean

```diff
@@ -45,11 +45,25 @@ I made a *heavy* use of `simp` lemmas, aiming to bring Laurent polynomials to th
 Any comments or suggestions for improvements is greatly appreciated!
 
 ##  Future work
-Lots is missing!  I would certainly like to show that `R[T;T⁻¹]` is the localization of `R[X]`
-inverting `X`.  This should be mostly in place, given `exists_T_pow` (which is part of PR #13415).
-(Riccardo) add inclusion into Laurent series.
-(Riccardo) giving a morphism (as `R`-alg, so in the commutative case)
-from `R[T,T⁻¹]` to `S` is the same as choosing a unit of `S`.
+Lots is missing!
+* (Riccardo) add inclusion into Laurent series.
+* (Riccardo) giving a morphism (as `R`-alg, so in the commutative case)
+  from `R[T,T⁻¹]` to `S` is the same as choosing a unit of `S`.
+* A "better" definition of `trunc` would be as an `R`-linear map.  This works:
```

Comment:
I am not sure, but I think this todo should start with `--`, otherwise it will end up in the official documentation.